makefile

```diff
@@ -9,36 +9,36 @@ python-dependencies:
 # Pipeline
 ## 1. Run: Build the part ontology from the intermediate Part Hierarchy files
 data/output/owl_component_files/part_ontology.owl: python-dependencies
-	python comp_loinc/build.py parts --schema-file ./model/schema/part_schema.yaml --part-directory ./data/part_files --output ./data/output/owl_component_files/part_ontology.owl
+	python3 comp_loinc/build.py parts --schema-file ./model/schema/part_schema.yaml --part-directory ./data/part_files --output ./data/output/owl_component_files/part_ontology.owl
```

Comment:
Bugfix: `python` -> `python3`: Fixes issue where `python` could be referring to python 2, which would cause syntax errors.